chain/processor.go

```diff
@@ -36,7 +36,6 @@ type ExecutionBlock struct {
 	// to preallocate memory
 	authCounts map[uint8]int
 	txsSet     set.Set[ids.ID]
-	sigJob     workers.Job
```

Comment:
Switch from setting on the block type and waiting later to returning and using later. This should be less error prone since if we don't use it again, we'll get a warning.